docs-devsite/functions.httpscallable.md

```diff
@@ -0,0 +1,33 @@
+Project: /docs/reference/js/_project.yaml
+Book: /docs/reference/_book.yaml
+page_type: reference
+
+{% comment %}
+DO NOT EDIT THIS FILE!
+This is generated by the JS SDK team, and any local changes will be
+overwritten. Changes should be made in the source code at
+https://github.com/firebase/firebase-js-sdk
+{% endcomment %}
+
+# HttpsCallable interface
```

Comment:
I'm puzzled about why this is new/green/add when it seems to have existed already in line 43 of public_types.ts, iiuc.  

In any case, it would be great to have clarity on whether we really mean "Google Cloud Functions" or whether we should stick with just "Cloud Functions" or even "Cloud Functions for Firebase."

Reply:
It's been a while since we made significant changes to this code, and it's possible the these type definition preceeds the devsite doc generation process I'm running here.

Reply:
I'll stick with Cloud Functions?

Reply:
SG, thanks!